CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 1.14.0 (May 29, 2024)
+
+ * [#4101](https://github.com/redhat-developer/vscode-openshift-tools/issues/4101) Project not listed if it has a display name annotation.
+ * [#4092](https://github.com/redhat-developer/vscode-openshift-tools/issues/4092) Trying to create a serverless function using the Spring Boot template causes the webview to fail.
+ * [#4086](https://github.com/redhat-developer/vscode-openshift-tools/issues/4086) Create an web view for supporting to create routes from the extension .
+ * [#4083](https://github.com/redhat-developer/vscode-openshift-tools/issues/4083) Helm chart browser broken.
+ * [#4064](https://github.com/redhat-developer/vscode-openshift-tools/pull/4064) Add a proposal message to change stdout max buffer length preference in case of 'ERR_CHILD_PROCESS_STDIO_MAXBUFFER' error appearance.
+ * [#4059](https://github.com/redhat-developer/vscode-openshift-tools/pull/4059) Get rid of using k8s extension provided 'kubectl' binary.
+ * [#3999](https://github.com/redhat-developer/vscode-openshift-tools/issues/3999) Make possible to work with projects/namespaces on clusters where the listing of projects is restricted.
+ * [#3990](https://github.com/redhat-developer/vscode-openshift-tools/issues/3990) Migrate Kubernetes extension commands.
+ * [#3987](https://github.com/redhat-developer/vscode-openshift-tools/issues/3987) App. Explorer "hangs" on extension loading if logged out from a cluster.
```

Comment:
I just wanted to mention that, even though this issue is fixed by the PR in "Migrate Kubernetes extension commands.", I think it's worth mentioning both since both are worth reporting.

Normally I would have expected a link only to the issue.

Reply:
Then I can suggest removing the link to the PR #4059 as it fixes the issue #3987, and add 
"Remove dependence on VS Code Kubernetes extension and use 'oc' binary instead of relying on 'kubectrl' binary provided by VS Code Kubernetes." to #3990

Reply:
The thing is, it's worth mentioning the "App. Explorer "hangs" on extension loading if logged out from a cluster." part. It's a separate issue that was fixed, and by only mentioning the technical aspect of extension migration, that part is left out.